Gemfile

```diff
@@ -3,6 +3,8 @@
 source 'http://rubygems.org'
 gemspec
 
+gem 'nokogiri', '~> 1.14', '>= 1.14.2'
```

Comment:
Nokogiri is a part of rails. So the version should only be updated in Gemfile.lock

Reply:
Good catch.